Classes/Domain/Service/CategoryImportService.php

```diff
@@ -80,43 +80,48 @@ public function import(array $importArray)
     {
         $this->logger->info(sprintf('Starting import for %s categories', count($importArray)));
 
-        // Sort import array to import the default language first
         foreach ($importArray as $importItem) {
+            $postPersistQueueItem = array();
             $category = $this->hydrateCategory($importItem);
 
             if (!empty($importItem['title_lang_ol'])) {
-                $this->postPersistQueue[$importItem['import_id']] = [
+                $postPersistQueueItem = array(
                     'category' => $category,
                     'importItem' => $importItem,
-                    'action' => self::ACTION_CREATE_L10N_CHILDREN_CATEGORY,
+                    'action' => array(self::ACTION_CREATE_L10N_CHILDREN_CATEGORY),
```

Comment:
Rather than turn this into an array I would simply create a copy of the $postPersistQueueItem with a different action, when this is needed. This yields two queued unique actions which seems cleaner - and it means this diff can be much smaller / change less business logic.